### ISSUE TITLE
Fullscreen button not working on WearOS (I am aware that Wear OS is not supported)

### ISSUE THREAD

**Issue by demirdegerli** [TeamNewPipe/NewPipe#10682]
### Checklist

- [X] I made sure that there are *no existing issues* - [open](https://github.com/TeamNewPipe/NewPipe/issues) or [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have read the [FAQ](https://newpipe.net/FAQ/) and my problem isn't listed.
- [X] I'm aware that this is a request for NewPipe itself and that requests for adding a new service need to be made at [NewPipeExtractor](https://github.com/TeamNewPipe/NewPipeExtractor/issues).
- [X] I have taken the time to fill in all the required details. I understand that the feature request will be dismissed otherwise.
- [X] This issue contains only one feature request.
- [X] I have read and understood the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md).

### Feature description

The app is working impressive on WearOS. Only issue is when I tap the fullscreen button on the video player, it doesn't go fullscreen. Could this be possible with some if checks? WearOS supports fullscreen but with a different API call as far as I know.

### Why do you want this feature?

The only fully-featured YouTube client is NewPipe on WearOS.

### Additional information

![Screenshot_20231222_215424_newpipe](https://github.com/TeamNewPipe/NewPipe/assets/67432156/a300a133-33ab-414e-852d-0aa3325ad4f4)
![Screenshot_20231222_215435_newpipe](https://github.com/TeamNewPipe/NewPipe/assets/67432156/6ecec9a1-e792-4850-91ca-937cfff2fb4d)
![Screenshot_20231222_215455_newpipe](https://github.com/TeamNewPipe/NewPipe/assets/67432156/e5dcbaee-6685-4e8f-9a12-e4a16cddeac0)
![Screenshot_20231222_215459_newpipe](https://github.com/TeamNewPipe/NewPipe/assets/67432156/645a6b84-0ee4-4aba-b3bf-df250032fbc9)
![Screenshot_20231222_215511_newpipe](https://github.com/TeamNewPipe/NewPipe/assets/67432156/7eaf96da-3fab-4108-9fea-7daf17652227)
![Screenshot_20231222_215521_newpipe](https://github.com/TeamNewPipe/NewPipe/assets/67432156/7a652f95-3407-4736-b7b3-f3235e3bb230)
![Screenshot_20231222_215524_newpipe](https://github.com/TeamNewPipe/NewPipe/assets/67432156/f8232979-714f-4da3-be01-f0ec4c495e79)

**Comment by demirdegerli** [comment]
> This is completely off-topic, but aren't you worried for your eyes? Squinting at tiny videos is not my idea of a good time.

It is useful while you don't have access to phone. I don't watch every video from my watch.

**Comment by demirdegerli** [comment]
> If there's a speaker, one could also simply listen to audio instead, I suppose. That seems useful.

Yes, I play music with it too. The best UI for a not-optimized app for Wear OS by far.

**Comment by WrittenSkate** [comment]
> This is completely off-topic, but aren't you worried for your eyes? Squinting at tiny videos is not my idea of a good time.


i know its like 3 years, but very odd and specific niche use case, I go to amusment parks a lot, like Canada's wonderland or cedar point. On peak summer days, lines can go up to 2 hours, even 3 sometimes. Specifically on top thrill 2, you cannot have anything in the line, including your phone, unless if it's a watch. So I'll have to just stand and wait for 2 hours doing nothing, but since i have a wearos watch, I can use that as my "phone" replacment while waiting in the top thrill 2 line, thus leading to watching youtube on it.
